pkgs/by-name/as/asfa/package.nix

```diff
@@ -0,0 +1,61 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  gawk,
+  help2man,
+  libssh2,
```

Comment:
```suggestion
```
This seems to no longer be used.

(btw I'm using `nil` as the language server, which marks unused variables, which is pretty useful)